nginx.conf

```diff
@@ -28,15 +28,18 @@ http {
   include /etc/nginx/conf.d/*.conf;
 
   server {
-    listen 80 default_server;
-    listen [::]:80 default_server;
+#     listen 80 default_server;
+#     listen [::]:80 default_server;
+    listen 8080 default_server;
+    listen [::]:8080 default_server;
     root /var/www/html;
     index index.html index.htm index.nginx-debian.html;
 
     server_name _;
 
     location / {
-      try_files $uri $uri/ =404;
+#       try_files $uri $uri/ =404;
+        try_files $uri $uri/ /index.html;
```

Comment:
so this is not exactly what I was looking for. Please read about $args nginx variable:
http://nginx.org/en/docs/http/ngx_http_core_module.html#var_args
or ($query_string)

Reply:
Maybe we can use this solution?
try_files $uri$args $uri$args/ $uri $uri/ /index.html =404;

Reply:
yes that is better! :)